features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/AppsAndKeys/TokenManager.jsx

```diff
@@ -88,6 +90,7 @@ class TokenManager extends React.Component {
         if (selectedApp) {
             this.appId = selectedApp.appId || selectedApp.value;
             this.application = Application.get(this.appId);
+            this.owner = selectedApp.owner;
```

Comment:
`this.owner = selectedApp.owner` is redundent since `selectedApp` coming from props ?

Reply:
resolved